apps/web/src/api/hooks/useWebhookSupportStatus.ts

```diff
@@ -0,0 +1,49 @@
+import { useQuery } from '@tanstack/react-query';
+import { ChannelTypeEnum } from '@novu/shared';
+
+import { getWebhookSupportStatus } from '../integration';
+import { IS_DOCKER_HOSTED, WEBHOOK_URL } from '../../config';
+import { useAuthController, useEnvController } from '../../hooks';
+
+export const useWebhookSupportStatus = ({
```

Comment:
the hook that verifies if the integration supports parsing events with the webhook